exchange/exchange.go

```diff
@@ -55,6 +55,7 @@ type exchange struct {
 	UsersyncIfAmbiguous bool
 	defaultTTLs         config.DefaultTTLs
 	privacyConfig       config.Privacy
+	eeaCountries        map[string]struct{}
```

Comment:
In a future PR, maybe we can refactor to have all GDPR related fields inside some privacy wrapper in order to simplify the `exchange` struct

Reply:
That's why I added privacyConfig. I didn't think to comment about adding this in there though.